Tests/Common/Data/TradeBarConsolidatorTests.cs

```diff
@@ -130,11 +130,11 @@ public void OneMinuteAlwaysFiresEveryTimeOnMinueDataExceptFirstPoint()
 
             consolidator.Update(new TradeBar {Close = 2m, Time = reference.AddMinutes(1)});
             Assert.IsNotNull(consolidated);
-            Assert.AreEqual(2, consolidated.Close);
+            Assert.AreEqual(1, consolidated.Close);
 
             consolidator.Update(new TradeBar {Close = 3m, Time = reference.AddMinutes(2)});
             Assert.IsNotNull(consolidated);
-            Assert.AreEqual(3, consolidated.Close);
+            Assert.AreEqual(2, consolidated.Close);
```

Comment:
I'm not sure this is the behavior we want here. I agree that when it swallows the first point it isn't good, but it looks like now we're always one time unit in the past.  Before this change I would be getting the newest data immediately, but now I'm a time step behind.

I wonder if we can fix this.